### ISSUE TITLE
Unable to select a specific frame

### ISSUE THREAD

**Issue by trapd00r** [jcs/ratpoison#3]
First off, thanks a ton for your work. I enjoy it everyday.

I've noticed an issue where at times a specific frame become un-focusable. See [this](https://youtu.be/rM9maXW7UDk) short video clip. As soon as I hsplit the left-most window, I can no longer select the frame at the right bottom.

(This issue doesn't exist in the main ratpoison tree).

**Comment by jcs** [comment]
Yeah I've run into this a few times and never had the time to debug it.  I usually just destroy the frames and re-split them to get around it while I'm working.

**Comment by trapd00r** [comment]
Yeah it wouldn't be that much of an issue if it only happened sometimes, but I can reproduce it every time I split my screen in 4 frames. Very weird.

**Comment by trapd00r** [comment]
The same issue exists in both the jcs and master branch, but I had an [old](https://github.com/trapd00r/catpoison/commits/master) checkout of your repo around where the issue doesn't exist: the last commit on that copy is from 2015-11-11.
